nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/AssetKind.java

```diff
@@ -26,6 +26,7 @@
 public enum AssetKind
 {
   HELM_INDEX(CacheControllerHolder.METADATA),
+  HELM_PROVENANCE(CacheControllerHolder.CONTENT),
```

Comment:
Is it really Content, not Metadata? It definitely looks like metadata.

Reply:
Yes. Provenance is content